src/formatDate.js

```diff
@@ -50,7 +50,45 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const [ a, b, c, separator ] = fromFormat;
+  const [ , , , newSeparator ] = toFormat;
```

Comment:
```suggestion
  const newSeparator = toFormat[3];
```

Reply:
Not resolved, just replace your code with `const newSeparator = toFormat[3];`